### DISCUSSION TITLE
Not able to access previous dsep community recordings

### DISCUSSION THREAD

**Discussion by mihirfossgen** [beckn/DSEP-Specification#59] [Q&A]
Dear Team,

 We are not able to access the previous community recordings please give permissions to the following email id's 

Mihir Rawal - [mihirr@fossgentechnologies.com]
Mahesh Musale - [maheshm@fossgentechnologies.com]
Kanchan Barskar - [barskark@fossgentechnologies.com]
Kuldeep Singh - [kuldeep@fossgentechnologies.com]

Thanks in advance.

**Comment by Anand-EkStep** [comment]
Could you please share some sample links which are not opening?

**Reply by mihirfossgen** [reply]
Hello Anand,

We are not able access this videos please check - 
Call 1 : https://drive.google.com/file/d/1fN-MgqGPWcWtiMS9lAC8w9CCgyFuoZcV/view?usp=drive_web
Call 2 : https://drive.google.com/file/d/1bguVMwOUsmoE7UndUyUiOPLFSXC71VuJ/view?usp=drive_web
Call 3 : https://drive.google.com/file/d/1bguVMwOUsmoE7UndUyUiOPLFSXC71VuJ/view?usp=sharing_eil_m&ts=62205c60
Call 4 : https://drive.google.com/file/d/1XyJuu0oB-yOqx4Hk15y52zWblLOLYfTi/view?usp=sharing_eip_m&ts=6228c5cb
Call 5 : https://drive.google.com/file/d/1SDK3LRMdL00qIb5RdSSnzsFQ9BRhU56Y/view?usp=sharing_eip_m&ts=623299b1
Call 6 : https://drive.google.com/file/d/1z8xcW2GuIYwwnq56_McLxQu9aEySj0sE/view?usp=sharing_eil_m&t
Call 10 : https://drive.google.com/file/d/1Bj6bQuu6m8Hl_rdTyh6JVfcoDTyaE0WK/view?usp=sharing_eil_m&ts=6269343c
Call 12 : https://drive.google.com/file/d/16OJyB8T6Dgjv2gJsT8Hc-3jbod9JU4Xq/view?ts=6284eabb

They are showing google drive permission is not given.

Thanks in advance.

**Reply by Anand-EkStep** [reply]
Thanks, the initial few recordings are still not open for access somehow. Even I do not have access to these.
I will ask Ravi to change the settings for these.
Thanks a lot for sharing the links.
@core-wg-admin

**Reply by Anand-EkStep** [reply]
Recordings should be accessible now.
